main.nf

```diff
@@ -109,6 +109,7 @@ if (params.genomes && params.genome && !params.genomes.containsKey(params.genome
 // Define these here - after the profiles are loaded with the iGenomes paths
 params.star_index = params.genome ? params.genomes[ params.genome ].star ?: false : false
 params.fasta = params.genome ? params.genomes[ params.genome ].fasta ?: false : false
+params.transcript_fasta = params.genome ? params.genomes[ params.genome ].transcript_fasta ?: false : false
```

Comment:
These parameters are there for files we have available on `AWS iGenomes`. We dont have any transcriptomes there at the moment so I dont think we should add this there. Also, seeing as its now quite easy to create the transcripts fasta from the `fasta` and `gtf` we should keep it that way if possible. Is there a reason you put it here?